includes/Assets.php

```diff
@@ -81,7 +81,7 @@ public function get_base_url( string $path ): string {
 	 *
 	 * @return array Array containing combined contents of "<$handle>.asset.php" and "<$handle>.chunks.php".
 	 */
-	protected function get_asset_metadata( string $handle ): array {
+	public function get_asset_metadata( string $handle ): array {
```

Comment:
Needed so you can access this method in our new class.

Reply:
That means we can update the `Assets` unit tests where this is called like this:

```php
$results = $this->call_private_method( $assets, 'get_asset_metadata', [ 'test_script' ] );
```

No longer need `call_private_method`